guide/src/python_from_rust.md

```diff
@@ -24,34 +24,39 @@ The example below shows a calling Python functions behind a `PyObject` (aka `Py<
 
 ```rust
 use pyo3::prelude::*;
-use pyo3::types::{PyDict, PyTuple};
+use pyo3::types::PyTuple;
 
-struct SomeObject;
-impl SomeObject {
-    fn new(py: Python) -> PyObject {
-        PyDict::new(py).to_object(py)
-    }
-}
-
-fn main() {
+fn main() -> PyResult<()> {
     let arg1 = "arg1";
     let arg2 = "arg2";
     let arg3 = "arg3";
 
     Python::with_gil(|py| {
-        let obj = SomeObject::new(py);
+        let fun: Py<PyAny> = PyModule::from_code(
+            py,
+    "def example(*args, **kwargs):
```

Comment:
Indentation is a bit strange here. I would propose, either completely to the left (8 chars leftwards), or completely within the normal Rust indentation (i.e. 8 chars rightwards). Or a separate constant outside of `main`...

Also you can use single-quotes on the Python side to avoid ugly escaping.